### PR TITLE
connect to mongodb,create database,collections and documents

### PR DESCRIPTION
**Pull request by bulaneNayo**

### CODE REVIEW

File: models/Question.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+const ObjectId = mongoose.Schema.Types.ObjectId;
+
+const QuestionInfoSchema = new mongoose.Schema(
+  {
+    quiestion_Text: {
+      type: String,
+      unique: true,
```
Comment (eliasbayona) [inline models/Question.js:8]: unique constraint not necessary

File: models/Game.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      type: Array,
+    },
+
+    player_answers: {
+      type: Array,
+    },
+
+    winner: {
+      type: ObjectId,
+      ref: "users",
```
Comment (eliasbayona) [inline models/Game.js:21]: Invalid ref, Ref are Case sensitive

File: models/Leaderboard.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+const LeaderboardSchema = new mongoose.Schema(
+  {
+
+    user_id:{
+      type:ObjectId,
+      ref: "users",
+    },
+    
+    winning_score:{
```
Comment (eliasbayona) [inline models/Leaderboard.js:12]: rename this to game if it is a ref to a game

File: models/User.js [hunk 4]
```diff
@@ -7,44 +7,53 @@ const userSchema = new mongoose.Schema(
...
+    refreshToken: { type: String },
+    accessToken: { type: String },
     role: {
       type: String,
-      default: Role.PLAYER
+      default: Role.PLAYER,
+    },
+
+    game_history:{
+      type: Boolean,
```
Comment (eliasbayona) [inline models/User.js:21]: invalid type, game_history should host a list of all games that the user has played

**Review by eliasbayona — CHANGES_REQUESTED** [review]

File: models/Leaderboard.js (hunk 3 above)
Comment (bulaneNayo) [reply models/Leaderboard.js:12]: i don't understand Elias, what exactly should i rename to game?

**Pull request merged**